metashare/api/jobs.py

```diff
@@ -247,21 +184,27 @@ def get_unsaved_changes(scratch_org):
         if unsaved_changes:
             scratch_org.latest_revision_numbers = new_revision_numbers
             scratch_org.unsaved_changes = unsaved_changes
```

Comment:
I just noticed this now... but why are we only doing this `if unsaved_changes`? It means that if we think there are changes, then we refresh and discover there aren't any... the org doesn't get updated to reflect that there actually aren't changes. 🤔